<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ONH_SRL_MUSCLE_SIM_2015Class</class>
 <widget class="QMainWindow" name="ONH_SRL_MUSCLE_SIM_2015Class">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ONH_SRL_MUSCLE_SIM_2015</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>231</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>Rotation</string>
    </property>
    <widget class="QSlider" name="xRotSlider">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>140</width>
       <height>20</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-180</number>
     </property>
     <property name="maximum">
      <number>180</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksAbove</enum>
     </property>
     <property name="tickInterval">
      <number>20</number>
     </property>
    </widget>
    <widget class="QSlider" name="zRotSlider">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>140</width>
       <height>20</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-180</number>
     </property>
     <property name="maximum">
      <number>180</number>
     </property>
     <property name="tracking">
      <bool>true</bool>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="invertedAppearance">
      <bool>false</bool>
     </property>
     <property name="invertedControls">
      <bool>false</bool>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksAbove</enum>
     </property>
     <property name="tickInterval">
      <number>20</number>
     </property>
    </widget>
    <widget class="QSlider" name="yRotSlider">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>140</width>
       <height>20</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-180</number>
     </property>
     <property name="maximum">
      <number>180</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksAbove</enum>
     </property>
     <property name="tickInterval">
      <number>20</number>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>X</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>60</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Y</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>90</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Z</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLCDNumber" name="lcdNumber">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>30</y>
       <width>51</width>
       <height>23</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 255, 0);
background-color: rgb(0, 0, 0);</string>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="smallDecimalPoint">
      <bool>false</bool>
     </property>
     <property name="segmentStyle">
      <enum>QLCDNumber::Flat</enum>
     </property>
    </widget>
    <widget class="QLCDNumber" name="lcdNumber_2">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>60</y>
       <width>51</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 255, 0);
background-color: rgb(0, 0, 0);</string>
     </property>
     <property name="segmentStyle">
      <enum>QLCDNumber::Flat</enum>
     </property>
    </widget>
    <widget class="QLCDNumber" name="lcdNumber_3">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>90</y>
       <width>51</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 255, 0);
background-color: rgb(0, 0, 0);</string>
     </property>
     <property name="segmentStyle">
      <enum>QLCDNumber::Flat</enum>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="onh_srl_muscle_sim_2015.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>xRotSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>99</x>
     <y>82</y>
    </hint>
    <hint type="destinationlabel">
     <x>205</x>
     <y>84</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>yRotSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber_2</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>99</x>
     <y>112</y>
    </hint>
    <hint type="destinationlabel">
     <x>205</x>
     <y>114</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>zRotSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber_3</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>99</x>
     <y>142</y>
    </hint>
    <hint type="destinationlabel">
     <x>205</x>
     <y>144</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
